.github/workflows/build_image.yml

```
@@ -8,12 +8,19 @@ on:
 
 jobs:
   build-and-push:
-    name: Build and push image to Google Artifact Registry
+    name: Build and push image to ${{ matrix.registry }} registry
```

```suggestion
    name: Build and push image to ${{ matrix.registry }} Google Artifact Registry
```
minor nit (just to distinguish this from GCR)